### PR TITLE
Dev to Staging 1.19.0

### PR DESCRIPTION
**Pull request by oskardrenske-ith**

### CODE REVIEW

**Review by erik-gbg — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,27 @@
 Changelog Application Jobsearch
 ===============================
+
```
Comment (erik-gbg) [inline CHANGELOG.md:3]: Fix changelog

File: common/settings.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import os
 
-API_VERSION = '1.18.1'
+API_VERSION = '1.18.2'
```
Comment (erik-gbg) [inline common/settings.py:3]: Fix version

**Review by erik-gbg — CHANGES_REQUESTED** [review]

**Review by lina1101 — APPROVED** [review]

**Review by halge — CHANGES_REQUESTED** [review]
i pause until i test memcache/apikeys.

**Review by erik-gbg — APPROVED** [review]

**Review by halge — COMMENTED** [review]
i need to test.

**Review by halge — APPROVED** [review]

**Pull request merged**